JenkinsFile

```diff
@@ -182,16 +183,19 @@ node {
     sh "mkdir -p ${root}/${artifactDirectory}"
     def tarball = "${appName}.tar.gz"
     withCredentials([file(credentialsId: "${env.JKS_FILE}", variable: 'JKS')]) {
+    withCredentials([file(credentialsId: "${env.GEOAXIS_JWT_CERT}", variable: 'GEOAXIS_JWT_CERT')]) {
       sh """
         [ -z "\$JKS" ] || mv \$JKS ${root}/src/main/resources/piazza.jks
+        [ -z "\$GEOAXIS_JWT_CERT" ] || mv \$JKS ${root}/src/main/resources/GEOAXIS_JWT_CERT.jks
```

Comment:
This is really a base64 encoded certificate, so it's going to have a `.pem` extension.

Reply:
Fixed